apps/test/unit/code-studio/components/pairing/StudentSelectorTest.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {expect} from '../../../../util/reconfiguredChai';
+import StudentSelector from '@cdo/apps/code-studio/components/pairing/StudentSelector';
+import {shallow} from 'enzyme';
+
+describe('StudentSelector', () => {
```

Comment:
I'd also recommend adding a test to check that `selected` and `selectable` are set correctly.

In enzyme, you can find an element by it's classname like this `expect(wrapper.find('.selected')).to.have.lengthOf(7)`

Reply:
✔️